packages/contentstack-import/src/utils/asset-helper.ts

```diff
@@ -434,6 +434,7 @@ function updateFileFields(
     for (let i = 0; i <= objekt.length; i++)
       updateFileFields(objekt[i], objekt, i, mappedAssetUids, matchedUids, unmatchedUids);
 
-    parent[pos] = _.compact(objekt);
+    // No need for _.compact() since you want to keep zero values
+    parent[pos] = _.filter(objekt, (value: any) => value !== undefined && value !== null);
```

Comment:
@aman19K since we are touching this code can we rename this variable from "objekt" to an appropriate name.

Reply:
done